src/styles/blocks/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
You have the normalize file for this purpose.

Reply:
got it, thank you)